hash/sgi_mips.xml

```diff
@@ -1572,7 +1614,21 @@ license:CC0
 		</part>
 	</software>
 
-	<!-- IndyZone -->
+	<!-- IndiZone -->
+
+	<software name="indizone_1">
+		<description>IndiZone</description>
```

Comment:
Duplicate description in "indizone_1" and "indizone".  Suggest you add the version number for each unique set in the long text description and not just in the **info name** tag.

Reply:
good catch, thanks.

Is it okay to rename existing entries as well while adding new ones? I would rename "indizone" to "indizone_1_1" (in the software/name tag, as well as in the diskarea/disk tag) while I'm at it

Reply:
Keep whatever form there is already for the softlist to keep everything consistant.  It would seem that all shortnames include the version as well in the examples I saw.